### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by humdingerb**

### CODE REVIEW

File: resources/views/help/user.twig [hunk 1]
```diff
@@ -13,18 +13,29 @@
 
 <p>{{ html_image('images/help/user-startpage.png', 'The start page with a list of all projects') }}</p>
 
-<p>After logging into Polyglot, just click on a project from the start page, and see if your language needs attention (i.e. its status is below 100%):</p>
+<p>After logging into Polyglot, the start page shows a list of all the hosted projects. After a project's name you may see a globe symbol, linking the it's website and a sheet icon linking to its bugtracker. Some developers also include the date of the next planned release, so translators can cram to get in their translations in time.<p>
```
Comment (KapiX) [inline resources/views/help/user.twig]: Unclosed tag at the end.

File: resources/views/help/user.twig [hunk 2]
```diff
@@ -13,18 +13,29 @@
...
-The <i>Translate</i> link takes you to the actual translation page:</p>
+<p>By default, only languages that are being actively translated are listed. If yours isn't among them, it means that nobody has started a translation yet: Click <i>Show all languages</i> to see it.</p>
+
+<p>If the progress of a translation is below 100% the menu <i>Continue</i> beside the progress bar is highlighted blue and a click brings you to a page with all still unfinished text blocks.<br />
+In the menu, the item <i>All text blocks</i> opens that page with all text blocks, the ones already translated and the ones still needing work.<br />
+<i>Export<i> lets you download the catkeys of that particular language, just like <i>Export all</i> at the bottom of the list of languages lets you download the catkeys of all languages. The downloaded file can then be edited in a text editor or <a href="https://depot.haiku-os.org/#!/pkg/catkeyseditor/haikuports/0/1/2/-/3/x86_gcc2">CatKeysEditor</a>, if you prefer that.</p>
```
Comment (KapiX) [inline resources/views/help/user.twig]: Unclosed tag.

**Comment by KapiX** [comment]
Thanks!

**Comment by humdingerb** [comment]
Damn! Sorry about those tags... I did look over it before creating the PR, but obviously missed it. Next time I'll transplant the the .twig into a HTML file and have Web+ warn me about that sort of thing.

**Pull request merged**